src/modules/user/domain/exceptions/exception.factory.ts

```diff
@@ -0,0 +1,21 @@
+import { DomainEventHandler } from '@common/domain/events/domain-events-handler';
+import DomainException from '@common/domain/exceptions/domain.exception';
+import { UserNotFoundException } from '@user/domain/exceptions/user-not-found.exceptions';
+
+export class ExceptionFactory {
+  static createException(type: string, details: string): void {
+    if (type === UserNotFoundException.USER_NOT_FOUND) {
+      /* empty */
```

Comment:
Te falta el manejo de la excepcion de dominio, te recomiendo utilizar la que estoy sugeriendo en código

```suggestion
           throw new ValidationDomainException(
        JSON.stringify({
          message: RoleDescriptionException.INVALID_ROLE_DESCRIPTION,
          details,
        }),
      );
```

Reply:
@2314-RAustin Listo, corregido.